### ISSUE TITLE
Remove static web content and make it configurable.

### ISSUE THREAD

**Issue by vnbang2003** [pythonindia/wye#429]
I think to make project adportable by other community we need to remove static conent and make it confirguable though static page.
This wil also help content team to easy change content.
I can be divided in multiple sub tickets
1) Home page
2) FAQ
3) Section on home page should also be confiration so users they can hide section not relevant to them.

**Comment by sourabhtk37** [comment]
Can you please elaborate on this a bit more?
do you want to have static content totally moved to it's own page and be included in the template using
`{% include "page.html" %}`.

**Comment by vnbang2003** [comment]
We should add file similar to common/base/constants.py to add all web page static content including logo path.
Each page should render content from this page.

Let me know if it make sense.

**Comment by sourabhtk37** [comment]
>Let me know if it make sense.

None whatsoever.

**Comment by vnbang2003** [comment]
Yes something similar to constants.py page or DB configurable table.
So it easy to change if any other community is willing to deploy this project.
